### DISCUSSION TITLE
Jupyter

### DISCUSSION THREAD

**Discussion by Boganabi** [DrAlzahraniProjects/csusb_fall2024_cse6550_team1#27] [General]
Discuss things related to Jupyter here!

**Comment by itsnotmik** [comment]
@Jagadeesh-ajjada 

I was doing some research on the Jupyter-AI being incompatible with Python 3.12 via your original link to jupyterlab/jupyter-ai#538 and it appears that this issue is resolved and compatibility with 3.12 was fixed. Was there a more recent issue with 3.12 that I am unaware of?
